### PR TITLE
remove win32/stdint.h, add CI build compatible with Windows XP

### PR DESCRIPTION
**Pull request by rfomin**
I found a portable [`stdint.h`](http://www.azillionmonkeys.com/qed/pstdint.h) and was able to build Woof with WATCOM C 11 and Borland C++ 5.5. But then I thought that this is idiocy and it’s better to remove the `stdint.h` nonsense altogether 😄 

Way more practical is MSCV 2017 version (the last one that supports Windows XP): 
[Woof-10.0.0-win32.zip](https://github.com/fabiangreffrath/woof/files/8935536/Woof-10.0.0-win32.zip) Maybe we can replace our 32-bit version with it. Fluidsynth isn't there yet, but it's an easy fix (they officially support Windows XP).

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
Cool, I am all for keeping things simple! Is it possible to integrate MSVC 2017 builds into our CI? Because else I wouldn't be able to do release builds anymore. What is the latest supported Windows version of our 32-bit MinGW build, btw?

**Comment by rfomin** [comment]
> Is it possible to integrate MSVC 2017 builds into our CI?

It looks like its possible to run v141_xp toolkit on windows-2019 image. I will try it.

> What is the latest supported Windows version of our 32-bit MinGW build, btw?

I can't find any official information about this. Mostly people use an old frozen installation that works and doesn't get updated.

**Comment by rfomin** [comment]
It works:
VirtualBox_winxp_20_06_2022_10_03_26
See Win32.zip artifact. See the Win32.zip artifact. No Fluidsynth yet, I think we should download the official version, as we do with SDL. An alternative is vcpkg, but it doesn't seem to support Windows XP.

**Comment by fabiangreffrath** [comment]
> Report: It works fine with FluidSynth on my Windows XP VM.

Awesome!

**Comment by fabiangreffrath** [comment]
Do we still need the `woof8.ico` icon variants if we target at least Win XP?

**Comment by rfomin** [comment]
> Do we still need the `woof8.ico` icon variants if we target at least Win XP?

`woof8.ico` is needed for old `resource.rc` compilers (VC6 and the like). We can remove them, but why? BTW VC6 build works fine on Windows 11.

**Comment by fabiangreffrath** [comment]
Okay, just asking. 😉

**Comment by rfomin** [comment]
> Okay, just asking. 😉

No, you're right. I need to clean up all the VC6 cruft.

File: CMakeLists.txt [hunk 1]
```diff
@@ -147,6 +142,23 @@ endif()
 find_package(SDL2_net)
 
 # FluidSynth backend
+if (NOT FluidSynth_DIR)
+    if (WIN32 AND MSVC AND CMAKE_SIZEOF_VOID_P STREQUAL 4)
```
Comment (fabiangreffrath) [inline CMakeLists.txt:146]: How about cross building?

Comment (rfomin) [reply CMakeLists.txt:146]: Fluidsynth releases two different archives for 64-bit and 32-bit versions. We need to update these downloads from time to time, so I thought less is better.

**Review by fabiangreffrath — APPROVED** [review]
Please merge if ready.

**Pull request merged**